doc/install/install-kubernetes.md

```diff
@@ -0,0 +1,120 @@
+# Install Kuadrant on a Kubernetes cluster
+
+> [!NOTE]  
+> You must perform these steps on each Kubernetes cluster where you want to use Kuadrant.
+
+
+## Prerequisites
+
+- Access to a Kubernetes cluster, with `kubeadmin` or an account with similar permissions
+
+## Procedure
+
+This guide will show you how to install Kuadrant onto a bare Kubernetes cluster.
+
+Alternatively, if you are looking instead for a way to set up Kuadrant locally to evaluate or develop, consider running the kind & Kubernetes [quickstart script](https://docs.kuadrant.io/latest/getting-started-single-cluster/).
+
+### Install Gateway API
+
+```bash
+kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.1.0/standard-install.yaml
+```
+
+### Install [OLM](https://olm.operatorframework.io/)
+
+```bash
+curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.23.1/install.sh | bash -s v0.23.1
+```
+
+### Install Istio as a Gateway API provider
+
+> [!NOTE]  
+> There are several ways to install Istio (via `istioctl`, Helm chart or Operator) - this is just an example for starting from a bare Kubernetes cluster.
+
+
+```bash
+curl -sSL https://istio.io/downloadIstio | ISTIO_VERSION=1.20.7 sh -
+./istio-1.20.7/bin/istioctl install --set profile=minimal -y
+./istio-1.20.7/bin/istioctl operator init
+kubectl apply -f https://raw.githubusercontent.com/Kuadrant/kuadrant-operator/main/config/dependencies/istio/istio-operator.yaml
+```
+
+### Install Kuadrant
+
+```bash
+kubectl create -f https://operatorhub.io/install/kuadrant-operator.yaml
```

Comment:
Missing `operators` namespace.
Should that have been created by olm installation, or manually by user?
```
$ kubectl create -f https://operatorhub.io/install/kuadrant-operator.yaml
Error from server (NotFound): error when creating "https://operatorhub.io/install/kuadrant-operator.yaml": namespaces "operators" not found

$ kubectl get ns
NAME              STATUS   AGE
default           Active   16m
istio-operator    Active   43s
istio-system      Active   7m21s
kube-node-lease   Active   16m
kube-public       Active   16m
kube-system       Active   16m
```

Reply:
I manually created the `operators` namespace,
then tried again.
Looks like something isn't setting up the kuadrant crds
```
kubectl -n kuadrant-system apply -f - <<EOF
→ apiVersion: kuadrant.io/v1beta1
→ kind: Kuadrant
→ metadata:
→   name: kuadrant
→ spec: {}
→ EOF
error: resource mapping not found for name: "kuadrant" namespace: "" from "STDIN": no matches for kind "Kuadrant" in version "kuadrant.io/v1beta1"
ensure CRDs are installed first
```

Reply:
OK, I think I know what happened here.
The install failed on a timeout the first time when creating the CRDs.
When I attempted to re-run, t said all the CRDs already existed, and exited the script.
The problem was that it should have continued and installed the olm deployments (the scrit has `-e` set)

I'd put this down to the olm install script not being idempotent enough

Reply:
added a CRD wait